File: CadastroProdutos-UI/src/app/models/Produtos.ts
import { DadosRegistro } from "./DadosRegistro";

export class Produtos {
    produtoId!:number;
    nomedoProduto!: string; 
    valordeVenda!:number; 
    foto!:File;
    usuarioId!:number; 
    usuario!: DadosRegistro
}

File: CadastroProdutos.BLL/Models/Produtos.cs
using CadastroProdutos.BLL.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CadastroProdutos.BLL
{
    public class Produtos
    {


        public int produtoId { get; set; }
        public string nomedoProduto { get; set; }
        public double valordeVenda { get; set; }
        public byte[] foto { get; set; }

        public string usuarioId { get; set; }
        public Usuario usuario { get; set; }


    }
}
